src/main/java/com/openlattice/aws/AwsS3Pod.java

```diff
@@ -52,7 +53,7 @@ public AmazonS3 awsS3() {
 
     public static AmazonS3 newS3Client( AmazonLaunchConfiguration awsConfig ) {
         AmazonS3ClientBuilder builder = AmazonS3ClientBuilder.standard();
-        builder.setRegion( Region.getRegion( awsConfig.getRegion().or( Regions.DEFAULT_REGION ) ).getName() );
+        builder.setRegion( Region.getRegion( awsConfig.getRegion().orElse( Regions.DEFAULT_REGION ) ).getName() );
```

Comment:
Getting a compile error on this line.

Reply:
Depends on https://github.com/geekbeast/rhizome-client/pull/32

Reply:
So whenever the build artifacts decide to propagate. . . . . 🤔

Reply:
Happy Thanksgiving :)